rpm/pacrunner.spec

```diff
@@ -1,7 +1,7 @@
 Name:       pacrunner
 
 Summary:    Proxy configuration daemon
-Version:    0.15
+Version:    0.18
 Release:    1
 License:    GPLv2+
 URL:        https://git.sailfishos.org/mer-core/pacrunner/
```

Comment:
Bonus points for updating this url to github from this already closed gitlab.

Reply:
Updated, thanks.